themes/classic/_dev/css/partials/_helpers.scss

```diff
@@ -0,0 +1,49 @@
+/*** HELPERS ***/
+
+._relative{
+  position: relative;
+}
+
+._dpt{
+  display: table;
+}
```

Comment:
Is it just the naming you don't like ? Like you'd prefere something like `margin-top-lg`,  `margin-vertical-sm` ?